eng/pipelines/templates/steps/analyze.yml

```diff
@@ -9,6 +9,16 @@ parameters:
 steps:
   - template: /eng/pipelines/templates/steps/analyze_dependency.yml
 
+  - template: /eng/common/pipelines/templates/steps/verify-links.yml
```

Comment:
I don't have a problem with moving this but I do don't like doing it based on a hidden dependency.  Is this just an optimization or is it a functional issue? I'd like to have this script work in isolation and not depend on analayze_dependency doing setup work for it.